server/weaverbird/backends/sql_translator/metadata.py

```diff
@@ -183,6 +187,10 @@ def remove(self) -> TableMetadata:
         self.delete = True
         return self
 
+    def remove_column_alias(self, column_name: str) -> TableMetadata:
+        self.columns[column_name].remove_alias()
```

Comment:
Not sure but, maybe we need to raise an error here when a column doesn't exist ?

Reply:
good one.